src/index.ts

```diff
@@ -305,7 +320,7 @@ async function appBoot() {
 		const clickedButtonIndex = dialog.showMessageBoxSync( {
 			message: __( 'Sync in progress' ),
 			detail: __(
-				'There’s a sync operation in progress. Quitting the app will abort that operation. Are you sure you want to quit?'
+				"There's a sync operation in progress. Quitting the app will abort that operation. Are you sure you want to quit?"
```

Comment:
Accident?

Reply:
yes, I guess the AI changed this randomly :)